gallery_dl/extractor/reddit.py

```diff
@@ -115,8 +115,10 @@ def items(self):
                         continue
                     if url[0] == "/":
                         url = "https://www.reddit.com" + url
-                    if url.startswith(
-                            "https://www.reddit.com/message/compose"):
+                    if (url.startswith(
+                            "https://www.reddit.com/message/compose") or
+                        url.startswith(
+                            "https://reddit.com/message/compose")):
```

Comment:
[`str.startswith()`](https://docs.python.org/3/library/stdtypes.html#str.startswith) supports multiple strings as arguments when you pack them as a tuple.
```suggestion
                    if url.startswith((
                        "https://www.reddit.com/message/compose",
                        "https://reddit.com/message/compose",
                    )):
```

Reply:
Even better.